runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -198,8 +198,13 @@ std::unique_ptr<onert::ir::Model> loadModel(const std::string filename,
 } // namespace
 
 nnfw_session::nnfw_session()
-  : _nnpkg{nullptr}, _coptions{}, _compiler_artifact{nullptr}, _execution{nullptr},
-    _kernel_registry{nullptr}
+  : _nnpkg{nullptr}, _coptions{}, _compiler_artifact{nullptr}, _execution{nullptr}, _kernel_registry
+{
+  nullptr
+}
+#ifdef ONERT_TRAIN
+, _training_step { 0 }
+#endif // ONERT_TRAIN
```

Comment:
```suggestion
```
@ragmani What about using member initialization?
It will remove this weird style and reduce `#ifdef ONERT_TRAIN` while both are identical.

Reply:
OK, I will change it.